Can't resolve 'react/jsx-runtime.js' when using with storybook-addon-designs

### Issue
Not sure if this repo is right for the issue, but when following https://storybook-addon-designs-f8z5gt8iw.vercel.app/?path=/story/docs-figma-figspec-readme--page, my storybook instance crashes with
```
ERROR in ./node_modules/@figspec/react/esm/es2015/index.js
Module not found: Error: Can't resolve 'react/jsx-runtime.js' in '/Users/pf1gura/Projects/Redacted/node_modules/@figspec/react/esm/es2015'
```

### Possible solutions that did not work
I already looked online and tries few solutions, bot nothing worked so far.
I tried adding those aliases to webpack config
```
config.resolve.alias['react/jsx-runtime'] = 'react/jsx-runtime.js';
config.resolve.alias['react/jsx-dev-runtime'] = 'react/jsx-dev-runtime.js';
```

### Dependencies
```
"storybook-addon-designs": "^6.0.0-alpha.1",
"@figspec/components": "^0.1.3",
"@figspec/react": "^0.1.4",
"@storybook/angular": "^6.1.11",
```

--- a/packages/react/src/index.tsx
+++ b/packages/react/src/index.tsx
@@ -6,6 +6,7 @@ import type {
   FigspecFileViewer as FigspecFileViewerElement,
 } from "@figspec/components";
 import {
+  createElement,
   forwardRef,
   useCallback,
   useEffect,
@@ -79,7 +80,9 @@ export const FigspecFrameViewer = forwardRef<
     },
     ref
   ) => {
-    const [refNode, setNode] = useState<FigspecFrameViewerElement | null>(null);
+    const [refNode, setNode] = useState<
+      (FigspecFrameViewerElement & HTMLElement) | null
+    >(null);
 
     useImperativeHandle<FigspecFrameViewerRef, FigspecFrameViewerRef>(
       ref,
@@ -87,14 +90,17 @@ export const FigspecFrameViewer = forwardRef<
       [refNode]
     );
 
-    const refCb = useCallback((node: FigspecFrameViewerElement | null) => {
-      if (node) {
-        setNode(node);
+    const refCb = useCallback(
+      (node: (FigspecFrameViewerElement & HTMLElement) | null) => {
+        if (node) {
+          setNode(node);
 
-        node.nodes = nodes;
-        node.renderedImage = renderedImage;
-      }
-    }, []);
+          node.nodes = nodes;
+          node.renderedImage = renderedImage;
+        }
+      },
+      []
+    );
 
     useEffect(() => {
       if (!refNode) return;
@@ -185,7 +191,9 @@ export const FigspecFileViewer = forwardRef<
     },
     ref
   ) => {
-    const [refNode, setNode] = useState<FigspecFileViewerElement | null>(null);
+    const [refNode, setNode] = useState<
+      (FigspecFileViewerElement & HTMLElement) | null
+    >(null);
 
     useImperativeHandle<FigspecFileViewerRef, FigspecFileViewerRef>(
       ref,
@@ -193,14 +201,17 @@ export const FigspecFileViewer = forwardRef<
       [refNode]
     );
 
-    const refCb = useCallback((node: FigspecFileViewerElement | null) => {
-      if (node) {
-        setNode(node);
+    const refCb = useCallback(
+      (node: (FigspecFileViewerElement & HTMLElement) | null) => {
+        if (node) {
+          setNode(node);
 
-        node.documentNode = documentNode;
-        node.renderedImages = renderedImages;
-      }
-    }, []);
+          node.documentNode = documentNode;
+          node.renderedImages = renderedImages;
+        }
+      },
+      []
+    );
 
     useEffect(() => {
       if (!refNode) return;

--- a/packages/react/tsconfig.json
+++ b/packages/react/tsconfig.json
@@ -3,7 +3,8 @@
     "target": "ES2015",
     "module": "ESNext",
     "declaration": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
+    "jsxFactory": "createElement",
     "lib": ["ESNext", "DOM"],
     "strict": true,
     "moduleResolution": "Node"

